packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -1,19 +1,41 @@
-import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
+import { makeResetStyles, makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { NavItemSlots, NavItemState } from './NavItem.types';
-import { typographyStyles } from '@fluentui/react-theme';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
 
 export const navItemClassNames: SlotClassNames<NavItemSlots> = {
   root: 'fui-NavItem',
   content: 'fui-NavItem__content',
+  icon: 'fui-NavItem__icon',
 };
 
+// These should match the constants defined in @fluentui/react-icons
+// This package avoids taking a dependency on the icons package for only the constants.
+const iconClassNames = {
+  filled: 'fui-Icon-filled',
+  regular: 'fui-Icon-regular',
+};
+
+const navItemTokens = { indicatorWidth: '4px', indicatorOffset: '-18px', indicatorHeight: '20px' };
+
 /**
  * Styles for the root slot
  */
-const useStyles = makeResetStyles({
+const useRootDefaultStyles = makeResetStyles({
   display: 'flex',
+  gap: tokens.spacingHorizontalL,
```

Comment:
For completeness. Same for `padding` below.

```suggestion
  gap: `${tokens.spacingHorizontalL} ${tokens.spacingVerticalL}`,
```

Reply:
updated